src/plugin/tasks/serverless/serverless-dynamodb-src-dy.json

```
@@ -0,0 +1,14 @@
+{
+  "target": "src/dynamodb/dy.js",
+  "format": "txt",
+  "strategy": "overwrite",
+  "snippets": [
+    "serverless-dynamodb/src/dy.js"
```

```suggestion
    "serverless-dynamodb/src/dynamodb/dy.js"
```
We always try to match the snippet directory with the target directory if possible

This change then also requires the file name to be changed `serverless-dynamodb-src-dynamodb-dy`

---

Yes, I removed that intentionally because it sounds redundant, will add the prefix again.